docs/source/model/signatures.rst

```diff
@@ -767,6 +776,48 @@ If your model has an optional input field, you can use below input_example as a
 
     print(infer_signature(input_example))
 
+If your model's output contains None values, these fields will be inferred as AnyType (since MLflow 2.19.0), for example:
+
+.. code-block:: python
+
+    from mlflow.models import infer_signature
+
+    data = [
+        {
+            "id": None,
+            "object": "chat.completion",
+            "created": 1731491873,
+            "model": None,
+            "choices": [
+                {
+                    "index": 0,
+                    "message": {
+                        "role": "assistant",
+                        "content": "MLflow",
+                    },
+                    "finish_reason": None,
+                }
+            ],
+            "usage": {
+                "prompt_tokens": None,
+                "completion_tokens": None,
+                "total_tokens": None,
+            },
+        }
+    ]
+
+    print(infer_signature(data))
+    # inputs:
+    # ['id': Any (optional), 'object': string (required), 'created': long (required), 'model': Any (optional),
+    # 'choices': Array({finish_reason: Any (optional), index: long (required),
+    # message: {content: string (required), role: string (required)} (required)}) (required),
+    # 'usage': {completion_tokens: Any (optional), prompt_tokens: Any (optional), total_tokens: Any (optional)} (required)]
+    # outputs:
+    # None
+    # params:
+    # None
```

Comment:
might not be necessary after simplifying the input data, but maybe we can use a multiline string and format the output for readability